code/modules/projectiles/guns/energy/special.dm

```diff
@@ -12,8 +12,6 @@
 	charge_cost = 300
 	max_shots = 10
 	projectile_type = /obj/item/projectile/ion
-	can_turret = 1

```

Comment:
Update your branch, because, you are reverting some changes here.

Reply:
woops. I forgot someone was doing turret changes.